src/neptune/attributes/atoms/float.py

```diff
@@ -45,5 +50,16 @@ def assign(self, value: typing.Union[FloatVal, float, int], *, wait: bool = Fals
         if not isinstance(value, FloatVal):
             value = FloatVal(value)
 
+        if is_unsupported_float(value.value):
+            warn_once(
+                message=f"WARNING: The value you're trying to log is a nonstandard float value ({str(value.value)}) "
```

Comment:
Remember to ask @normandy7 to review the warning content.

Reply:
It is copied from previous PR, which she reviewed 😄 but asked her to doublecheck, that everything is correct